index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="stylesheet.css" />
+  </head>
+  <body>
+    <nav class="nav_links">
+      <img src="geo1.png" alt="cool logo" />
+      <div class="burger">
+        <div class="line1"></div>
+        <div class="line2"></div>
+        <div class="line3"></div>
+      </div>
+    </nav>
+    <section class="hero">
+      <h2>hello</h2>
+      <div class="logo">
+        Shaheer <br />
+        Chaudry
+      </div>
+      <p class="greeting">
+        Hello! I am a software developer! I can help you build a product,
+        feature or website. Taka a look of my works. <br />
+        If you like what you see and have a project you need coded, don’t
+        hesitate and contact me.
+      </p>
+
+      <button type="button">Lets Connect</button>
```

Comment:
![image](https://user-images.githubusercontent.com/38088818/164913849-99f2b2e9-5d7b-44f8-b798-6cf657cdf8bd.png)
Your `let's connect` button must be aligned center and should be the same width as the description text like in the figma design.

Figma Button:
![image](https://user-images.githubusercontent.com/38088818/164913808-6b61b3f1-e734-4128-8d31-1cfa3c2e98e5.png)